pkg/asset/asset_test.go

```diff
@@ -26,11 +26,11 @@ func (a *persistAsset) Generate(Parents) error {
 
 type writablePersistAsset struct {
 	persistAsset
-	files []*File
+	FileList []*File
```

Comment:
`files` seems like the correct naming.

Reply:
It clashes with the interface function Files. Had to change one of them. Do you prefer to change the function name?

Reply:
hmm, lets keep it as is then.